featuretools/primitives/standard/transform/postal/one_digit_postal_code.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Categorical, PostalCode
+
+from featuretools.primitives.base import TransformPrimitive
+from featuretools.utils.gen_utils import Library
+
+
+class OneDigitPostalCode(TransformPrimitive):
+    """Returns the one digit prefix from a given postal code.
+
+    Description:
+        For a list of postal codes, returns the one digit prefix for a given postal code.
+
+    Examples:
+        >>> one_digit_postal_code = OneDigitPostalCode()
+        >>> one_digit_postal_code(['92432', '34514']).tolist()
+        ['9', '3']
+    """
+
+    name = "one_digit_postal_code"
+    input_types = [ColumnSchema(logical_type=PostalCode)]
+    compatibility = [Library.PANDAS, Library.DASK, Library.SPARK]
```

Comment:
If we are including Dask and Spark here, I think we should include tests with Dask and Spark inputs to explicitly test those cases. Also true for `TwoDigitPostalCode`.